accounts/libaccounts.go

```diff
@@ -354,26 +359,27 @@ func unlockAccounts(acnts utils.AccountsWithWeight) {
 }
 
 // uncompressUnits returns the uncompressed value of the units if compressFactor is provided
-func uncompressUnits(units *utils.Decimal, cmprsFctr int) (tU *utils.Decimal) {
+func uncompressUnits(units *utils.Decimal, cmprsFctr int, acntChrg *utils.AccountCharge, uf map[string]*utils.UnitFactor) (tU *utils.Decimal) {
```

Comment:
Please pass here directly the UnitFactor. If missing out of map, it should panic, same as AccountCharge.